flink-connector-kafka/src/test/resources/log4j2-test.properties

```diff
@@ -18,7 +18,7 @@
 
 # Set root logger level to OFF to not flood build logs
 # set manually to INFO for debugging purposes
-rootLogger.level = OFF
+rootLogger.level = INFO
```

Comment:
do we really need that or is it just a leftover?

Reply:
left over!